src/openforms/registrations/contrib/camunda/apps.py

```diff
@@ -1,12 +1,22 @@
+from decimal import Decimal
+
 from django.apps import AppConfig
 from django.utils.translation import gettext_lazy as _
 
+from django_camunda.utils import TYPE_MAP as CAMUNDA_TYPE_MAP
+
 
 class CamundaApp(AppConfig):
     name = "openforms.registrations.contrib.camunda"
     label = "registrations_camunda"
     verbose_name = _("Camunda registration plugin")
 
     def ready(self):
+        patch_django_camunda()
```

Comment:
Why not update django_camunda? This doesn't seem OF specific?

Reply:
as discussed during standup - mapping a Python `Decimal` with fixed precision to a Java Double (with floating point precision errors) is probably not guaranteed to be correct, but a best effort we can do.

At some point we'll probably include it in django-camunda with a disclaimer or a setting to enable/override it.